test/suites/app-rfid-llrp-inventory/main_test.go

```diff
@@ -15,16 +15,15 @@ const (
 )
 
 func TestMain(m *testing.M) {
-	start := time.Now()
-
-	log.Println("[SETUP]")
-
 	// start clean
 	utils.SnapRemove(nil,
 		appRfidLlrpSnap,
 		"edgexfoundry",
 	)
 
+	log.Println("[SETUP]")
+	start := time.Now()
```

Comment:
I moved this to after snap removals, to have logs collected from installation onwards.